test.js

```diff
@@ -11,6 +11,7 @@ const fixture = '🦄';
 
 test.beforeEach(t => {
 	t.context.conf = new Conf({cwd: tempy.directory()});
+	t.context.config = new Conf({cwd: tempy.directory(), accessPropertiesByDotNotation: false});
```

Comment:
It's weird that this is called `config` and the other one `conf`. Be more specific.